src/features/MentorPage/components/MentorList/index.tsx

```diff
@@ -13,17 +14,21 @@ type Props = {
 
 const MentorList: React.FC<Props> = ({ setVisibleCard, mentors }) => {
   const { isMobile } = useGetLayoutMode();
+  const isMentorsEmpty: boolean = mentors.length === 0;
 
   return (
-    <CardsList isMobile={isMobile} data-testid="mentor-cards-container">
-      {mentors.map(mentor => (
-        <ListCard
-          key={mentor.buddyId}
-          mentor={mentor}
-          setVisibleCard={setVisibleCard}
-        />
-      ))}
-    </CardsList>
+    <>
```

Comment:
This is a nitpick, but maybe I would not render the empty CardsList-divs if no mentors are found. Instead I would return a ternary 

`isMentors ? <MentorsList /> : <EmptyList />`

Reply:
yeah this is better, thanks